src/NServiceBus.Core.Analyzer.Tests.Common/ForwardCancellationToken/ForwardFromPipelineTests.cs

```diff
@@ -107,9 +107,6 @@ public class TestTimeout {}";
                 code = code.Replace("public Task", "public override Task");
             }
 
-            TestContext.WriteLine($"Source Code for test case: {type.FullName}:");
-            TestContext.WriteLine(code);
```

Comment:
Not needed in any case, now that the analyzer testing infrastructure takes care of doing this.